stream-chat-android-client/src/main/java/io/getstream/chat/android/client/extensions/ContextExtensions.kt

```diff
@@ -0,0 +1,16 @@
+package io.getstream.chat.android.client.extensions
+
+import android.content.ComponentName
+import android.content.Context
+import android.content.Intent
+
+internal fun Context.sendImplicitBroadcast(intent: Intent) {
```

Comment:
Android prohibits implicit intents for broadcast receivers register in the manifest. This workaround transforms implicit intent to explicit.